### PR TITLE
Add Aishell-4 recipe

### PR DESCRIPTION
**Pull request by yhliangElaina**

### CODE REVIEW

File: examples/aishell4/s0/run.sh [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+# use average_checkpoint will get better result
+average_checkpoint=true
+decode_checkpoint=$dir/final.pt
+average_num=30
+decode_modes="attention_rescoring"
+
+. tools/parse_options.sh || exit 1;
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    echo "stage -1: Data Download"
```
Comment (robin1001) [inline examples/aishell4/s0/run.sh:50]: use two space to indent.

**Review by robin1001 — APPROVED** [review]

**Comment by peggyxpxu** [comment]
hi，I found a lot of errors when running local/prepare_data.sh. Does this code work ？

**Comment by robin1001** [comment]
@yhliangElaina

**Comment by 1633347510** [comment]
i also want to make this model work on aishell4, but i met some errors in the stage 0(prepare_data.sh): some important files are missing,where can access these files? thanks a lot !

**local/prepare_data.sh: line 75: local/text_normalize.pl: No such file or directory
local/prepare_data.sh: line 75: local/text_format.pl: No such file or directory**

**Pull request merged**